lib/transcription.js

```diff
@@ -0,0 +1,82 @@
+const {createClient} = require("@deepgram/sdk");
```

Comment:
refactor deepgram specific stuff somehow.  One idea is a superclass transcriber that has subclass implementations for each vendor